cdap-client/src/main/java/io/cdap/cdap/client/ScheduleClient.java

```diff
@@ -183,6 +184,25 @@ public String getStatus(ScheduleId scheduleId) throws IOException, Unauthenticat
     return responseObject.get("status");
   }
 
+  /**
+   * Update schedules which were suspended between startTimeMillis and endTimeMillis.
+   *
+   * @param namespaceId the namespace in which to restart schedules
+   * @param startTimeMillis lower bound in millis of the update time for schedules (inclusive)
+   * @param endTimeMillis upper bound in millis of the update time for schedules (exclusive)
+   */
+  public void reEnableSuspendedSchedules(NamespaceId namespaceId, long startTimeMillis, long endTimeMillis)
+    throws IOException, UnauthorizedException, UnauthenticatedException, NotFoundException {
+    String path =
+      String.format("schedules/re-enable?start-time-millis=%d&end-time-millis=%d", startTimeMillis, endTimeMillis);
+    URL url = config.resolveNamespacedURLV3(namespaceId.getNamespaceId(), path);
+    HttpResponse response = restClient.execute(HttpMethod.PUT, url, config.getAccessToken(),
+                                               HttpURLConnection.HTTP_NOT_FOUND);
+    if (HttpURLConnection.HTTP_NOT_FOUND == response.getResponseCode()) {
+      throw new NotFoundException(namespaceId);
+    }
```

Comment:
Should also check for other type of error response code. Should only return normally if the response status is 200.

Reply:
Nevermind, the RESTClient.execute() method checked and thrown already.